clientgui/SkinManager.cpp

```diff
@@ -19,6 +19,8 @@
 #pragma implementation "SkinManager.h"
 #endif
 
+#include <sys/types.h>
```

Comment:
In this particular case these headers are used for linux-only builds.
Please add the corresponding macro defined to exclude them from any other builds